### ISSUE TITLE
Feature extraction

### ISSUE THREAD

**Issue by TABREZ-96** [prejin2310/Inverse-Cooking#1]
can u kindly add feature extraction and show how it is working and also add accuracy

**Comment by prejin2310** [comment]
Refer these links :
1. http://pic2recipe.csail.mit.edu/
2. https://scontent.ftrv1-1.fna.fbcdn.net/v/t39.8562-6/10000000_316621373573041_5913521806368798043_n.pdf?_nc_cat=110&ccb=1-7&_nc_sid=ad8a9d&_nc_ohc=jYOV_EOvZqYAX_UooYq&_nc_ht=scontent.ftrv1-1.fna&oh=00_AfDZBZnsObZ57YNv5g5utHgAjK8Z-1PggMG-BaG-QhcBZA&oe=64261D5A

**Comment by TABREZ-96** [comment]
Yeah i have referred if you have any knowledge about the project if you
could help kindly contact me.
It might take your 2min of time if u don't mind
 thank you
